operator-common/src/main/java/io/strimzi/operator/common/Util.java

```diff
@@ -556,12 +556,20 @@ public static String hashStub(String toBeHashed)   {
      * @return              First 8 characters of the SHA-1 hash
      */
     public static String hashStub(byte[] toBeHashed)   {
+        byte[] digest = getSHA1(toBeHashed);
+        return String.format("%040x", new BigInteger(1, digest)).substring(0, 8);
+    }
+
+    /**
+     * Get a SHA-1 hash of the provided byte array
+     * @param toBeHashed    Byte array for which the hash will be returned
+     * @return              SHA-1 hash
+     */
+    public static byte[] getSHA1(byte[] toBeHashed) {
```

Comment:
It is not a getter. So you can maybe call it `sha1Digest()`? Assuming this method is really needed.

Reply:
Done